views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl

```diff
@@ -134,6 +134,48 @@
                 </table>
             </div>
         </div>
+    {elseif $input.type == 'shoppingfeed_marketplace_switch_list'}
+        <div class="form-group">
+            <label class="control-label col-lg-3">
+                <span class="label-tooltip">{l s='Send invoices to Shopping Feed' mod='shoppingfeed'}</span>
+            </label>
+            <div class="col-lg-9">
+                <div class="alert alert-info">
+                    {l s='This option allows you to send invoices to Shopping Feed for transmission to the marketplace.' mod='shoppingfeed'}
+                </div>
+                <table class="table">
+                    <thead>
+                    <tr>
+                        <th>
+                            {l s='Market place' mod='shoppingfeed'}
+                        </th>
+                        <th>
+                            {l s='Send invoices' mod='shoppingfeed'}
+                        </th>
+                    </tr>
+                    </thead>
+
+                    <tbody>
+                    {foreach from=$input.marketplaces item=marketplace}
+                        <tr>
+                            <td>{$marketplace->getName()}</td>
```

Comment:
escape

Reply:
fixed

Reply:
OK in order_status_syncro_178.tpl but not in order_status_syncro.tpl

Reply:
fixed